crates/exchange-rate-oracle/src/lib.rs

```diff
@@ -177,15 +191,14 @@ pub mod pallet {
     // The pallet's dispatchable functions.
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        /// Sets the exchange rate.
+        /// Feeds data from the oracles, e.g., the exchange rates. This function
+        /// is intended to be API-compatible with orml-oracle.
         ///
         /// # Arguments
         ///
-        /// * `exchange_rate` - i.e. planck_per_satoshi = dot_per_btc * (10**10 / 10**8)
-        /// This is the same unit that is stored in the ExchangeRate storage item.
-        #[pallet::weight(<T as Config>::WeightInfo::set_exchange_rate())]
-        #[transactional]
-        pub fn set_exchange_rate(origin: OriginFor<T>, exchange_rate: UnsignedFixedPoint<T>) -> DispatchResult {
+        /// * `values` - a vector of (key, value) pairs to submit
+        #[pallet::weight(<T as Config>::WeightInfo::feed_values(values.len() as u32))]
+        pub fn feed_values(origin: OriginFor<T>, values: Vec<(OracleKey, T::UnsignedFixedPoint)>) -> DispatchResult {
```

Comment:
Personally I'm not a huge fan of the API but I get that we want to be closely compatible with orml-oracle.